client/src/store/state.ts

```diff
@@ -138,4 +142,9 @@ export const initialStoreState: State = {
   lobbyWaitingUsers: 0,
   lobbyClientJoinedQueue: false,
   lobbyReceivedInvitation: false,
-};
+
+  //TUTORIAL TRADING
```

Comment:
These tutorial variables are not really tutorial variables so they should be under a trade request key. Maybe something like

```
ui: {
  tradeRequest: {
    partner: string,
    give: ResourceInventory
    receive: ResourceInventory
  }
}
```

any other component local state should live under the UI key